cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunBuildToolsTask.java

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
Let's update the license header if applicable.

Reply:
Let's ignore this since the PR is already merged

Reply:
The initial commits are from 2023, hence the year 2023 in the header.